src/main/java/de/retest/recheck/RecheckImpl.java

```diff
@@ -215,8 +217,7 @@ String getDifferencesErrorMessage( final TestReplayResult finishedTestResult,
 				+ "You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).\n" //
 				+ "\n" //
 				+ numChecks + " check(s) in '" + suiteName + "' found the following difference(s):\n" //
-				+ allDiffs //
-				+ insertedDeletedDiffs.toString();
+				+ allDiffs + insertedDeletedDiffs.toString();
```

Comment:
I think there should be a "\n", since the printer does not ensure a newline at the end.

Reply:
Makes sense, but let me first write a test for it.